Proteomics/ModificationWithLocation.cs

```diff
@@ -64,20 +64,11 @@ public override bool Equals(object o)
             return m == null ? false :
 
                base.Equals(m)
-               && (this.accession == null && m.accession == null
```

Comment:
Are there modifications that are the same in UniProt with different accession numbers?